ruby-challenges.rb

```diff
@@ -75,4 +75,5 @@ def word_reader (string)
 # 1. input a method that will take in a string and decide if its a palindrome
 # 2. output a string that says whether the word is or isnt
 # 3.created the method and made if else statements with strings in them
-# 4.
\ No newline at end of file
+# 4.
+
```

Comment:
Good job overall!  Don't forget to cover both upper and lower case in question 2.  Be sure your output matches the expected output.  You're right there - you've got this!